BlazorScheduler/Components/Scheduler.razor.cs

```diff
@@ -15,7 +15,10 @@ public partial class Scheduler : IAsyncDisposable
         [Parameter] public RenderFragment Appointments { get; set; } = null!;
         [Parameter] public RenderFragment<Scheduler>? HeaderTemplate { get; set; }
         [Parameter] public RenderFragment<DateTime>? DayTemplate { get; set; }
-
+        [Parameter] public string? RootDaysStyle { get; set; }
+        [Parameter] public string? RootAppointmentOverflowStyle { get; set; }
+        [Parameter] public string? RootDayClass { get; set; }
```

Comment:
It's hard to tell what the difference is between `RootDaysStyle` and `RootDayStyle`. Could we get some more descriptive names?